src/quacc/recipes/newtonnet/core.py

```diff
@@ -0,0 +1,399 @@
+"""
+Core recipes for the NewtonNet code
+"""
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Literal
+
+import covalent as ct
+import numpy as np
+from ase.atoms import Atoms
+from ase.optimize.optimize import Optimizer
+from ase.units import _c, fs
+from ase.vibrations.data import VibrationsData
+from monty.dev import requires
+
+try:
+    from sella import IRC, Sella
+except:
+    Sella = None
+from quacc import SETTINGS
+from quacc.schemas.ase import (
+    summarize_opt_run,
+    summarize_run,
+    summarize_thermo_run,
+    summarize_vib_run,
+)
+from quacc.util.calc import run_ase_opt, run_calc
+from quacc.util.thermo import ideal_gas
+
+try:
+    from newtonnet.utils.ase_interface import MLAseCalculator as NewtonNet
+except ImportError:
+    NewtonNet = None
+
+
+def get_hessian(atoms):
```

Comment:
For all "internal" functions (ones that the user isn't expected to use), please rename them with `_` in the front, e.g. `_get_hessian` and `_add_stdev_and_hess`.

Reply:
Done with this one. Renamed the functions. Added docstrings and type hints.